include/openmc/mesh.h

```diff
@@ -243,72 +268,115 @@ class RectilinearMesh : public StructuredMesh
   std::vector<std::vector<double>> grid_;
 };
 
-#ifdef DAGMC
-
+// Abstract class for unstructured meshes
 class UnstructuredMesh : public Mesh {
 
 public:
-  UnstructuredMesh() = default;
-  UnstructuredMesh(pugi::xml_node);
-  ~UnstructuredMesh() = default;
+  // Constructors
+  UnstructuredMesh() {};
+  UnstructuredMesh(pugi::xml_node node);
+  UnstructuredMesh(const std::string& filename);
 
-  void bins_crossed(const Particle& p,
-                    std::vector<int>& bins,
-                    std::vector<double>& lengths) const override;
+  // Methods
+public:
```

Comment:
Can we remove this second usage of `public` and just nest the non-constructor methods with the constructors? I don't see other places where we use the `public` access two times for methods

Reply:
Yeah totally. Accidentally left that in there.